README.md

```diff
@@ -70,7 +70,7 @@ docker run \
     -e NB_INPUT_SRID=2249 \
     -e NB_BOUNDARY_BUFFER=11000 \
     -v /vagrant/data/:/data/ \
-    pfb
+    pfb-analysis
```

Comment:
Not from a change in this PR, but since we're editing the README it would be good to change the example commands to reflect that the output SRID is now calculated automatically into the appropriate meter-based UTM projection, so this boundary buffer that was intended as 2 miles in feet is now 6.8 miles in meters.